lib/activities/output.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+var fs = require("fs"),
+    path = require("path"),
+    url = require("url");
+
+var debug = require("debug"),
+    mkdirp = require('mkdirp');
+
+var s3Output = require("./s3_output");
+
+var log = debug("swfr:upload");
+
+module.exports = function(outputUri, done, fn) {
+  if(outputUri.indexOf("s3://") === 0) {
+    return s3Output(outputUri, done, fn);
+  } 
+
+  // Local output
```

Comment:
It doesn't.  Might be nice for consistency though.

`if (uri.protocol === "file:") {
  var filename;

  if (uri.host) {
    // handle relative paths (e.g. file://./whatever.tif)
    filename = path.join(process.cwd(), uri.host) + uri.pathname;
  } else {
    // absolute paths
    filename = uri.host + uri.pathname;
  }
} else {
  return fn(new Error("Unsupported output protocol: " + outputUri));
}